### PR TITLE
Jetty 9 fix

### PR DESCRIPTION
**Pull request by MarianaTorresTorres**

### CODE REVIEW

File: .theia/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (KevinWorkman) [inline .theia/settings.json:1]: I think you can add these settings.json files to your .gitignore file, they don't need to be in your repo. But they aren't hurting anything, so it's up to you!

**Review by KevinWorkman — APPROVED** [review]

File: .theia/settings.json (hunk 1 above)
Comment (MarianaTorresTorres) [reply .theia/settings.json:1]: So, do I just put .theia in the .gitignore file? Or do I put .theia/settings.json?

Comment (MarianaTorresTorres) [reply .theia/settings.json:1]: Oh also should I put the .vscode stuff in the .gitignore as well?

Comment (KevinWorkman) [reply .theia/settings.json:1]: Yeah I think that sounds right to me!

**Pull request merged**